src/entity/SourcePostModeration.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Post } from './posts';
+import { Source } from './Source';
+import { User } from './user';
+
+export enum SourcePostModerationStatus {
+  Approved = 'approved',
+  Rejected = 'rejected',
+  Pending = 'pending',
+}
+
+@Entity()
+export class SourcePostModeration {
+  @PrimaryColumn({ type: 'text' })
+  id: string;
```

Comment:
Isn't this just generated uuid?

Reply:
This is how we define the primary column in most of our entities, is it not? 🤔

Example post:

```
export class Post {
  @PrimaryColumn({ type: 'text' })
  id: string;
```

Reply:
Post is kind of legacy old. it also generates it's own ID which is odd.
You can use something like this maybe?
https://github.com/dailydotdev/daily-api/blob/bf166a5159205b029bf32450000cdb47086791d2/src/entity/DevCard.ts#L23

Reply:
Alright, will update